meteor-slingshot/services/cloudflare-r2.js

```diff
@@ -0,0 +1,179 @@
+const crypto = Npm.require('crypto');
+const url = Npm.require('url');
+
+Slingshot.CloudflareR2 = {
+  accessId: 'AccessKeyId',
+  secretKey: 'SecretAccessKey',
+
+  directiveMatch: {
+    bucket: String,
+    accountId: String,
+    endpoint: String,
+    cdn: String,
+    region: Match.Optional(String),
+    AccessKeyId: String,
+    SecretAccessKey: String,
+    key: Match.OneOf(String, Function),
+    expire: Match.Where(function (expire) {
+      check(expire, Number);
+      return expire > 0;
+    }),
+  },
+
+  directiveDefault: {
+    region: 'auto',
+    expire: 5 * 60 * 1000,
+  },
+
+  getContentDisposition: async function (method, directive, file, meta) {
+    var getContentDisposition = directive.contentDisposition;
+
+    if (!_.isFunction(getContentDisposition)) {
+      getContentDisposition = function () {
+        var filename = file.name && encodeURIComponent(file.name);
```

Comment:
why `var` here?

Reply:
This part of the code I got from here:

https://github.com/quavedev/meteor-packages/blob/819efa007f418fe0ed15d991cafa1547eff08b2e/meteor-slingshot/services/aws-s3.js#L68

Updated there as well.